File: RePlan/src/main/java/com/replan/api/dto/FixedScheduleRequest.java
package com.replan.api.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class FixedScheduleRequest {
    private Long userId;
    private String title;
    private String startTime;
    private String endTime;
    private boolean repeat;
    private String repeatDay;
}

File: app/src/main/java/com/example/replan/FixedScheduleRequest.kt
package com.example.replan

import com.google.gson.annotations.SerializedName

data class FixedScheduleRequest(
    @SerializedName("title") val title: String,        // 일정 제목
    @SerializedName("dayOfWeek") val dayOfWeek: String, // 요일 (월, 화, 수...)
    @SerializedName("startTime") val startTime: String, // 시작 시간 (예: "14:00")
    @SerializedName("endTime") val endTime: String      // 종료 시간 (예: "15:00")
)